src/components/Dashboard/cells/sponsors.js

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+
+
+import amadeus from '../../../assets/sponsors/amadeus.png';
+import bp from '../../../assets/sponsors/bp.png';
+import c1 from '../../../assets/sponsors/c1.png';
+import caterpillar from '../../../assets/sponsors/caterpillar.png';
+import citadel from '../../../assets/sponsors/citadel.png';
+import facebook from '../../../assets/sponsors/facebook.png';
+import fulcrum from '../../../assets/sponsors/fulcrum.png';
+import google from '../../../assets/sponsors/google.png';
+import imc from '../../../assets/sponsors/imc.png';
+import jackson from '../../../assets/sponsors/jackson.png';
+import mirus_research from '../../../assets/sponsors/mirus_research.png';
+import mozilla from '../../../assets/sponsors/mozilla.png';
+import npm from '../../../assets/sponsors/npm.svg';
+import nvisia from '../../../assets/sponsors/nvisia.png';
+import optum from '../../../assets/sponsors/optum.png';
+import particle from '../../../assets/sponsors/particle.png';
+import rc from '../../../assets/sponsors/rc.png';
+import schlum from '../../../assets/sponsors/schlum.png';
+import snl from '../../../assets/sponsors/snl.png';
+import two_sigma from '../../../assets/sponsors/two_sigma.png';
+import zeit from '../../../assets/sponsors/zeit.svg';
+
+//https://stackoverflow.com/questions/2450954/how-to-randomize-shuffle-a-javascript-array/2450976#2450976
+function shuffle(array) {
```

Comment:
You could add this to Array.prototype if you wanna be cool

Reply:
The webpack linter said I shouldn't add anything to Array.prototype.
The exact message: 
Array prototype is read only, properties should not be added  no-extend-native